sdk/spring/azure-spring-servicebus/src/main/java/com/azure/spring/servicebus/core/queue/ServiceBusQueueTemplate.java

```diff
@@ -60,13 +60,17 @@ protected void internalSubscribe(String name, Consumer<Message<?>> consumer, Cla
             this.checkpointConfig, payloadType, consumer, this.messageConverter) {
 
             @Override
-            protected String buildCheckpointFailMessage(Message<?> message) {
-                return String.format(MSG_FAIL_CHECKPOINT, message, name);
+            protected void buildCheckpointFailMessage(Message<?> message, Throwable t) {
```

Comment:
If this method is used to log information, should we change the method names accordingly?